tools/ssm/ssm_create.sh

```diff
@@ -17,31 +17,37 @@ set -eu
 PKG_NAME=$1
 REL_VERSION=$2
 S3_BUCKET=$3
+REGION=$4
 
-output=$(aws ssm list-documents --filter Key=Owner,Values=Self Key=DocumentType,Values=Package Key=Name,Values=${PKG_NAME})
+output=$(aws --region ${REGION} ssm list-documents --filter Key=Owner,Values=Self Key=DocumentType,Values=Package Key=Name,Values=${PKG_NAME})
 echo ${output}
 status=$(echo ${output} | python3 -c "import sys, json; print('empty') if len(json.load(sys.stdin)['DocumentIdentifiers']) == 0 else print('exist')")
 if [[ ${status} == "empty" ]]
 then
-    aws ssm create-document \
+    aws --region ${REGION} ssm create-document \
         --name "${PKG_NAME}" \
         --content file://build/packages/ssm/manifest.json \
         --attachments Key="SourceUrl",Values="https://s3.amazonaws.com/${S3_BUCKET}" \
         --version-name ${REL_VERSION} \
         --document-type Package
 else
-    output=$(aws ssm update-document \
-        --name "${PKG_NAME}" \
-        --content file://build/packages/ssm/manifest.json \
-        --attachments Key="SourceUrl",Values="https://s3.amazonaws.com/${S3_BUCKET}" \
-        --version-name ${REL_VERSION} \
-        --document-version "\$LATEST")
+    len=$(aws --region ${REGION} ssm list-document-versions --name "${PKG_NAME}" | python3 -c "import sys, json; print(len([ver for ver in json.load(sys.stdin)['DocumentVersions'] if ver['VersionName'] == '${REL_VERSION}']))")
```

Comment:
Since we already have the python script, can we use python for ssm_create as well, it is shelling out to python3 for json anyway (btw: you can also use `jq` but I think using python for all of them reduce maintain workload.  You can install boto like [this link](https://github.community/t/help-with-aws-cli/17054/2)

Reply:
Agree. The logic of ssm_create.sh is simple and it is trivial to rewrite in python. However I'd like to use pre-installed AWS CLI instead of installing boto since AWS CLI is enough to handle the situation.

Reply:
Rewrite script in python with boto3 as you suggested.